README.md

```diff
@@ -61,8 +97,8 @@ In order to execute this example on your local machine you will have to checkout
 
 ```
 brew upgrade pyenv
-pyenv install 3.6.0
-pyenv global 3.6.0
+pyenv install 3.8.13
```

Comment:
I have updated python version to be used because on M1 3.6.0 needs additional patches (not supported directly)